<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoServerDialog</class>
 <widget class="QDialog" name="InfoServerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: '#0d3d4c';
}</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 700 30pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1px;
	color: '#e3e3e3';
}</string>
   </property>
   <property name="text">
    <string>Multi Chat</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lineLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>201</width>
     <height>4</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: '#101014';
	border-radius: 2px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipTextBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 4px solid '#101014';
	font: 700 12pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 2px;
	color: '#e3e3e3';
	background-color: '#0d3d4c';
	padding-left: 13px;
}

QLineEdit::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="portTextBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 4px solid '#101014';
	font: 700 11pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 2px;
	color: '#e3e3e3';
	background-color: '#0d3d4c';
	padding-left: 13px;
}

QLineEdit::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 700 18pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1px;
	color: '#e3e3e3';
}</string>
   </property>
   <property name="text">
    <string>Users</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 700 18pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1px;
	color: '#e3e3e3';
}</string>
   </property>
   <property name="text">
    <string>Muted Users</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 700 18pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1px;
	color: '#e3e3e3';
}</string>
   </property>
   <property name="text">
    <string>Banned Users</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 700 18pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1px;
	color: '#e3e3e3';
}</string>
   </property>
   <property name="text">
    <string>Messages</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="usersCountBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>173</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 4px solid '#101014';
	font: 700 10pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 2px;
	color: '#e3e3e3';
	background-color: '#0d3d4c';
	padding-left: 13px;
}

QLineEdit::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="mutedUsersBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>223</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 4px solid '#101014';
	font: 700 10pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 2px;
	color: '#e3e3e3';
	background-color: '#0d3d4c';
	padding-left: 13px;
}

QLineEdit::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="bannedUsersBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>273</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 4px solid '#101014';
	font: 700 10pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 2px;
	color: '#e3e3e3';
	background-color: '#0d3d4c';
	padding-left: 13px;
}

QLineEdit::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="messagesBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>323</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 4px solid '#101014';
	font: 700 10pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 2px;
	color: '#e3e3e3';
	background-color: '#0d3d4c';
	padding-left: 13px;
}

QLineEdit::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
